src/Scripting/ConfigBindings.cpp

```diff
@@ -0,0 +1,84 @@
+#include "ConfigBindings.h"
+
+#include <Library/Logger/Logger.h>
+#include <Engine/Engine.h>
+
+#include <functional>
+#include <string>
+
+ConfigBindings::ConfigBindings(const sol::state_view &solState) : _solState(solState) {
+}
+
+sol::table ConfigBindings::getBindingTable() {
+    if (!_bindingTable) {
+        _bindingTable = _solState.create_table_with(
+            "setConfig", sol::overload(
+                [this](std::string_view section, std::string_view config, std::string_view value) { return setConfigValue(section, config, value); },
+                [this](std::string_view config, std::string_view value) { return setConfigValue(config, value); }
+            ),
+            "getConfig", sol::overload(
+                [this](std::string_view section, std::string_view config) { return getConfigValue(section, config); },
+                [this](std::string_view config) { return getConfigValue(config); }
```

Comment:
I have a feeling you should make the functions `static` and drop `this` from the capture list. I would also go a bit further, rename the functions `getConfigValue2` and `getConfigValue1` (or some other fitting names) and then this gets turned into `sol::overload(&ConfigBindings::getConfigValue1, &ConfigBindings::getConfigValue2)`.